### ISSUE TITLE
-o json Output is different from API

### ISSUE THREAD

**Issue by maboehm** [ionos-cloud/ionosctl#249]
## Description
The JSON output generated by addding the flag `-o json` to any command is different from the output produced when calling an endpoint directly. E.g.
```
$ ionosctl datacenter list -o json | jq 'keys'
[
  "Resources"
]
$ curl -s https://api.ionos.com/cloudapi/v6/datacenters --user "$IONOS_USERNAME:$IONOS_PASSWORD" | jq keys
[
  "_links",
  "href",
  "id",
  "items",
  "limit",
  "offset",
  "type"
]
```
I don't necessarily mind that the metadata is lost, but the fact that the top level key `items` gets renamed to `Resources` is very inconvenient - e.g. I would have expected that a `jq` query works the same whether I use the API directly or `ionosctl -o json`

## Expected behavior
JSON fields in the API Response are not renamed

## Environment

Ionosctl version: 
```
v6.5.1
```

## References
This is caused by the way resulting lists are passed to the Printer, which specifies the JSON representation to be `Resources` https://github.com/ionos-cloud/ionosctl/blob/master/pkg/printer/result.go#L132

**Comment by avirtopeanu-ionos** [comment]
Will be fixed by #348

**Comment by avirtopeanu-ionos** [comment]
#348 has been merged into master, now all API fields will be correctly shown in `-o json`

```
 % i dc ls -o json | jq keys
[
  "_links",
  "href",
  "id",
  "items",
  "limit",
  "offset",
  "type"
]
```

You can expect this to be included in the next release!
